pyartcd/pyartcd/pipelines/buils_sync.py

```diff
@@ -0,0 +1,92 @@
+        _LOGGER.info("Running command: %s", cmd)
+        _, stdout, _  = await exectools.cmd_gather_async(cmd)
+        async with aiofiles.open(f"${ns}.backup.yaml", "w") as f:
+            await f.write(stdout.read())
+        tar.add(f"${ns}.backup.yaml")
+
+        cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} get secret/release-upgrade-graph -n ${ns} -o yaml"
+        _LOGGER.info("Running command: %s", cmd)
+        _, stdout, _  = await exectools.cmd_gather_async(cmd)
+        async with aiofiles.open(f"${ns}.release-upgrade-graph.backup.yaml", "w") as f:
+            await f.write(stdout.read())
+        tar.add(f"${ns}.release-upgrade-graph.backup.yaml")
+    tar.close()
+
+
+@build_sync.command('trigger', short_help='Reads streams.yml and mirrors out ART equivalent images to api.ci.')
+@click.option('--version', metavar='VERSION', required=True, help='OCP version')
+@pass_runtime
+@click_coroutine
+async def trigger_release(version, dry_run):
+    cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} -n ocp tag registry.access.redhat.com/ubi8 {version}-art-latest:trigger-release-controller"
+    _LOGGER.info("Running command: %s", cmd)
+    await exectools.cmd_assert_async(cmd)
+    time.sleep(10)
+    cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} -n ocp tag {version}-art-latest:trigger-release-controller -d"
+    _LOGGER.info("Running command: %s", cmd)
+    await exectools.cmd_assert_async(cmd)
+
+
+@build_sync.command('update', short_help='Reads streams.yml and mirrors out ART equivalent images to api.ci.')
+@click.option('--doozer_data_path', metavar='Doozer_data_path', required=True, help='doozer_data_path dir')
+@click.option('--doozer_working_dir', metavar='Doozer_working_dir', required=True, help='doozer_working dir')
+@click.option('--build_version', metavar='BUILD_VERSION', required=True, help='OCP version')
+@click.option('--assembly', metavar='Assembly', required=True, help='assembly args can be version or stream')
+@click.option('--imagelist', metavar='Imagelist', required=False, default=None, help='(Optional) Limited list of images to sync, for testing purposes')
+@click.option('--excludeArches', metavar='excludeArches', required=False, help='(Optional) List of problem arch(es) NOT to sync (aarch64, ppc64le, s390x, x86_64)')
+@click.option('--dry_run', required=False, is_flag=True, help='mock running oc command')
+@click.option('--emergency_ignore_issues', required=False, is_flag=True, help='Ignore all issues with constructing payload. Do not use without approval.')
+@click.option('--publish', required=False, is_flag=True, help='Publish release image(s) directly to registry.ci for testing')
+@click.option('--debug', required=False, is_flag=True, help='Run "oc" commands with greater logging')
+@pass_runtime
+@click_coroutine
+def update_imagestream(runtime, imagelist, doozer_data_path, doozer_working_dir, build_version,assembly, excludeArches, dry_run, emergency_ignore_issues, publish, debug):
```

Comment:
Taking this verb as an example, this is just a simple wrapper of Doozer verb `release:gen-payload` without any additional logic.
I think the value of pyartcd here is the orchestration layer for our existing toolings (elliott, doozer, etc). If a pipeline just needs to invoke a single elliott/doozer verb, we don't have to wrap it into artcd because it is a waste of effort. But if we need multiple invocations, we orchestrate here.